Source/Scene/QuadtreePrimitive.js

```diff
@@ -994,10 +994,10 @@ define([
         var error = (maxGeometricError * height) / (distance * sseDenominator);
 
         if (frameState.fog.enabled) {
-            error = error - CesiumMath.fog(distance, frameState.fog.density) * frameState.fog.sse;
+            error -= CesiumMath.fog(distance, frameState.fog.density) * frameState.fog.sse;
         }
 
-        return error;
+        return error /= frameState.screenSpaceErrorPixelRatio;
```

Comment:
Should be `return error / frameState.screenSpaceErrorPixelRatio;`. Or better, match `screenSpaceError2D`:

```js
        error /= frameState.screenSpaceErrorPixelRatio;

        return error;
```